client/src/components/MultiJurisdictionAudit/Setup/Review/useSampleSizes.ts

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from 'react'
+import { toast } from 'react-toastify'
+import { api } from '../../../utilities'
+import { ISampleSizeOption } from '../../../../types'
+
+export interface IStringSampleSizeOption {
+  size: string
+  key: string
+  prob: number | null
+}
+
+export interface IStringSampleSizeOptions {
+  [key: string]: IStringSampleSizeOption[]
```

Comment:
i think it's kind of confusing to call this `key` because `key` refers to something different in `IStringSampleSizeOption`. I think `contestId` would be clearer. same feedback for the other types below

Reply:
@jonahkagan I actually forgot you could use something other than `key` there, haha.